packages/core/src/components/forms/_form-group.scss

```diff
@@ -86,6 +119,12 @@ Styleguide pt-form-group
       margin: 0 $pt-grid-size 0 0;
       line-height: $pt-input-height;
     }
+
+    // not ideal but baseline doesn't align text perfectly
+    // so we have to align it to the label manually
+    .pt-control {
+      margin-top: $pt-grid-size * 1.2;
```

Comment:
That didn't work, the `.pt-control` markup doesn't vertically centers. I went back to `baseline`, it's literally the right tool for the job, I suspect the alignment problems are browser issues, specifically regarding text baseline _within_ an input. I can live with that if you guys can too, check out the new preview